src/main/java/Duke/Tasks/TasksList.java

```diff
@@ -0,0 +1,94 @@
+package Duke.Tasks;
+
+import Duke.Exceptions.InvalidCommandFormatException;
+import Duke.Exceptions.TaskAlreadyMarkedException;
+import Duke.Exceptions.TaskListEmptyException;
+import Duke.Exceptions.TaskNumberOutOfBoundsException;
+import Duke.Exceptions.TaskNumberNotNumberException;
+
+import java.util.ArrayList;
+
+public class TasksList {
+    protected static ArrayList<Task> tasksList = new ArrayList<>();
+
+    public int getTasksListSize() {
+        return tasksList.size();
+    }
+
+    public void printHorizontalLine() {
+        System.out.println("____________________________________________________________");
+    }
+
+    public void printAddTaskText(Task task) {
+        System.out.println("Got it. I've added this task:");
+        System.out.println(task.taskStatusWithDescriptionText());
+        System.out.println("Now you have " + getTasksListSize() + " tasks in the list.");
+        printHorizontalLine();
+    }
+
+    public void addToTasksList(Task task) {
+        tasksList.add(task);
+    }
+
+    public void printTaskList() throws TaskListEmptyException {
+        if (getTasksListSize() == 0) {
+            throw new TaskListEmptyException();
+        } else {
+            System.out.println("Here are the tasks in your list:");
+            for (int i = 0; i < getTasksListSize(); i++) {
+                System.out.println((i+1) + "." + tasksList.get(i).taskStatusWithDescriptionText());
```

Comment:
Consider adding space between operators.
```suggestion
                System.out.println((i + 1) + "." + tasksList.get(i).taskStatusWithDescriptionText());
```
> Operators should be surrounded by a space character.

https://se-education.org/guides/conventions/java/index.html#layout